### PR TITLE
Feature/load custom disease records

### PR DESCRIPTION
**Pull request by vgainullin**

### CODE REVIEW

File: phenopy/d2p.py [hunk 1]
```diff
@@ -86,10 +54,88 @@ def load(phenotype_annotations_file, hpo_network, alt2prim, logger=None):
...
+
+    # TODO: do we need phenotype_to_diseases?
+    return disease_records, phenotype_to_diseases
+
+
 def frequency_converter(hpoa_frequency):
     """convert the frequency column from the hpoa file to a float"""
     if 'HP:' in hpoa_frequency:
-        return hpo_id_to_float.get(hpoa_frequency, '')
+        return hpo_id_to_float.get(hpoa_frequency, 0.5)
```
Comment (arvkevi) [inline phenopy/d2p.py:138]: If no frequency is given, we assume it's half? That seems strong. Can you put a `#TODO` here so we can come back and discuss the best default?

File: phenopy/d2p.py [hunk 2]
```diff
@@ -98,4 +144,4 @@ def frequency_converter(hpoa_frequency):
     elif '%' in hpoa_frequency:
         return float(hpoa_frequency.strip('%')) / 100
     # return 0.5 by default
-    return None
+    return 0.5
```
Comment (arvkevi) [inline phenopy/d2p.py:147]: same as above

**Review by arvkevi — CHANGES_REQUESTED** [review]

File: phenopy/d2p.py [hunk 3]
```diff
@@ -4,78 +4,46 @@
...
             reader.fieldnames[0] = 'DatabaseID'
 
-            disease_to_phenotypes = dict()
-            phenotype_to_diseases = dict()
+            records = []
+
             for row in reader:
                 # phenotype term id
                 # convert alternate phenotype id to primary
-                term_id = row['HPO_ID'] if row['HPO_ID'] not in alt2prim else alt2prim[row['HPO_ID']]
```
[the remark this answers is not in the document]
Comment (vgainullin) [reply phenopy/d2p.py:32]: see line 120. this is still there, I will remove the comment

File: phenopy/d2p.py [hunk 4]
```diff
@@ -86,10 +54,88 @@ def load(phenotype_annotations_file, hpo_network, alt2prim, logger=None):
...
+def read_custom_annotation_file(custom_annotation_file_path, hpo_network, logger=None):
+    try:
+        with open(custom_annotation_file_path, 'r') as tsv_fh:
+            reader = csv.reader(tsv_fh, delimiter='\t')
+
+            records = []
+            for row in reader:
+                # phenotype term id
+                # convert alternate phenotype id to primary
+                term_id, disease_accession, freq = row
```
[the remark this answers is not in the document]
Comment (vgainullin) [reply phenopy/d2p.py:66]: see line 120. this is still there, I will remove the comment

**Pull request merged**